app/packages/looker/src/overlays/base.ts

```diff
@@ -67,6 +67,7 @@ export interface Overlay<State extends Partial<BaseState>> {
   draw(ctx: CanvasRenderingContext2D, state: State): void;
   isShown(state: Readonly<State>): boolean;
   field?: string;
+  label?: BaseLabel;
```

Comment:
This is not true for `Classifications`. Is the `CoordinateOverlay` type is sufficient for the purpose of this PR?
```suggestion
```